### DISCUSSION TITLE
How to achieve continuous dialogue?

### DISCUSSION THREAD

**Discussion by VashonHu** [eternnoir/pyTelegramBotAPI#2153] [Q&A]
That is, the user asks the robot to perform certain tasks through a series of sequential instructions.

**Comment by denatuiumsaccharide** [comment]
> That is, the user asks the robot to perform certain tasks through a series of sequential instructions. @VashonHu 


The bot has to ask for an input via a message that bot sends to the user, and after the user responded then you can assign that user's text to a varible in your code and do what ever you want with it. This process can be **sequential**, meaning that you can ask for many things from a user and then do something with those user inputs... .

for example see:
[bot.register_next_step_handler](https://github.com/eternnoir/pyTelegramBotAPI/blob/master/examples/step_example.py)
